libs/wingsdk/src/shared-aws/domain.ts

```diff
@@ -17,15 +17,48 @@ export class Domain extends cloud.Domain {
   constructor(scope: Construct, id: string, props: cloud.DomainProps) {
     super(scope, id, props);
 
-    const values = getPlatformSpecificValues(
-      this,
-      "iamCertificate||acmCertificateArn",
-      "hostedZoneId"
+    const registrar = App.of(scope).platformParameterRegistrar;
```

Comment:
(optional) Shorten name maybe
```suggestion
    const parameters = App.of(scope).platformParameters;
```